backend/corpora/lambdas/upload_failures/slack.py

```diff
@@ -11,22 +11,24 @@
 logger = logging.getLogger(__name__)
```

Comment:
We should relocate all of this code to common/utils so you don't need to import the the upload_failure directory.